packages/combobox/src/Combobox.tsx

```diff
@@ -0,0 +1,923 @@
+          break;
+        }
+      }
+    }
+  };
+
+  /**
+   *
+   * Global Event Handler
+   *
+   */
+  // Global backdrop click handler
+  const handleBackdropClick = ({ target }: MouseEvent) => {
+    const isChildFocused =
+      menuRef.current?.contains(target as Node) ||
+      comboboxRef.current?.contains(target as Node) ||
+      false;
+    setOpen(isChildFocused);
+  };
+  useEventListener('mousedown', handleBackdropClick);
+
+  return (
+    <ComboboxContext.Provider
+      value={{
+        multiselect,
+        darkMode,
+        size,
+        withIcons,
+        chipTruncationLocation,
+        chipCharacterLimit,
+        inputValue,
+      }}
+    >
+      <div
+        className={cx(
+          comboboxParentStyle({ darkMode, size, overflow }),
+          className,
+        )}
+        {...rest}
+      >
+        <div>
+          {label && (
+            <Label id={labelId} htmlFor={inputId}>
+              {label}
+            </Label>
+          )}
+          {description && <Description>{description}</Description>}
+          {/* TODO - add error state message */}
+        </div>
+
+        <InteractionRing
+          className={interactionRingStyle}
+          disabled={disabled}
+          color={interactionRingColor({ state, darkMode })}
+        >
+          {/* Disable eslint: onClick sets focus. Key events would already have focus */}
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+          <div
+            ref={comboboxRef}
+            role="combobox"
+            aria-expanded={isOpen}
+            aria-controls={menuId}
+            aria-owns={menuId}
+            tabIndex={-1}
+            className={comboboxStyle}
+            onClick={setInputFocus}
+            onFocus={handleInputFocus}
+            onKeyDown={handleKeyDown}
+            onTransitionEnd={handleTransitionEnd}
+            data-disabled={disabled}
+            data-state={state}
+          >
+            <div
+              ref={inputWrapperRef}
+              className={inputWrapperStyle({ overflow, isOpen, selection })}
+            >
+              {renderedChips}
+              <input
+                aria-label={ariaLabel ?? label}
+                aria-autocomplete="list"
+                aria-controls={menuId}
+                aria-labelledby={labelId}
+                ref={inputRef}
+                id={inputId}
+                className={inputElementStyle}
+                placeholder={placeholderValue}
+                disabled={disabled ?? undefined}
+                onChange={handleInputChange}
+                value={inputValue}
+              />
+            </div>
+            {renderedInputIcons}
+          </div>
+        </InteractionRing>
+
+        {state === 'error' && errorMessage && (
+          <div className={errorMessageStyle}>{errorMessage}</div>
+        )}
+
+        {/******* /
+          *  Menu  *
+          / *******/}
+        <Popover
```

Comment:
Currently, clicking on an option in the combobox is causing the `:active` pseudo-class of the button to appear.

Reply:
Also, in single select mode, clicking on the currently selected option should probably close the dropdown. (Sorry, not sure exactly what lines this comment would be relevant on).

Reply:
>  clicking on an option in the combobox is causing the :active pseudo-class of the button to appear.

I don't think it's the `:active` pseudo-class—I think it's removing the `:focus` pseudo-class

Reply:
Might be wrong on the issue here 😄 regardless, the symptom is there.